Block/Checkout/Success.php

```diff
@@ -130,7 +138,7 @@ public function getMultibancoData()
     {
         $result = [];
         if (!empty($this->getOrder()->getPayment()) &&
-            !empty($this->getOrder()->getPayment()->getAdditionalInformation('comprafacil.entity'))
+            strcmp($this->getOrder()->getPayment()->getAdditionalInformation('paymentMethodType'), 'multibanco') === 0
```

Comment:
Check if the `paymentMethodType` is not empty
```suggestion
            !empty($this->getOrder()->getPayment()->getAdditionalInformation('paymentMethodType')) &&
            strcmp($this->getOrder()->getPayment()->getAdditionalInformation('paymentMethodType'), 'multibanco') === 0
```